packages/kit/src/runtime/client/router.js

```diff
@@ -242,7 +242,7 @@ export class Router {
 
 			if (incorrect) {
 				info.path = has_trailing_slash ? info.path.slice(0, -1) : info.path + '/';
-				history.replaceState({}, '', `${info.path}${location.search}`);
+				history.replaceState({}, '', `${this.base}${info.path}${location.search}`);
```

Comment:
Can you drop this change from this PR? It's unrelated so should be its own PR

Reply:
Oops, I was fixing another issue, my bad.